src/Sylius/Bundle/FlowBundle/spec/Process/Context/ProcessContextSpec.php

```diff
@@ -46,7 +46,7 @@ function it_initializes(
     ) {
         $process->getScenarioAlias()->shouldBeCalled();
         $storage->initialize(Argument::type('string'))->shouldBeCalled();
-        $process->getOrderedSteps()->shouldBeCalled()->willReturn(array($previousStep, $currentStep, $nextStep));
+        $process->getOrderedSteps()->shouldBeCalled()->willReturn([$previousStep, $currentStep, $nextStep]);
```

Comment:
As you change this file anyway, please remove all calls to `shouldBeCalled()` if there are calls to the `willReturn()` like on this line.